interfaces/Portalicious/src/assets/projects-empty.svg

```diff
@@ -0,0 +1,29 @@
+<svg
+  width="54"
+  height="49"
+  viewBox="0 0 54 49"
+  fill="none"
+  xmlns="http://www.w3.org/2000/svg"
+>
+<path
+    d="M24.9704 47.0479L7.49058 39.8493C5.83937 39.1693 5.83937 36.8307 7.49058 36.1507L26.5117 28.3174C27.0049 28.1143 27.5588 28.1166 28.0503 28.3238L46.6291 36.1571C48.2597 36.8446 48.2598 39.1554 46.6291 39.8429L29.5862 47.0286C28.1118 47.6503 26.45 47.6572 24.9704 47.0479Z"
+    stroke="black"
+    stroke-width="2"
+  />
+<path
+    d="M13.8485 36.0537L14.7977 36.3682L13.8485 36.0537C13.4661 37.2076 13.8263 38.6477 15.1512 39.2352C16.9516 40.0336 20.6477 41.1231 27.4153 41.1231C34.1323 41.1231 37.8235 40.0498 39.6387 39.2532C40.9966 38.6572 41.3485 37.1737 40.9253 36.003L29.7525 5.09097C28.7845 2.41278 24.9791 2.46387 24.0834 5.16708L13.8485 36.0537Z"
+    fill="white"
+    stroke="#0F1218"
+    stroke-width="2"
+  />
+<path
+    d="M21.7539 14.7693C21.7539 14.7693 23.7231 16.2462 27.1693 16.2462C30.6154 16.2462 32.5847 14.7693 32.5847 14.7693"
+    stroke="black"
+    stroke-width="2"
+  />
+<path
+    d="M17 26C17 26 20.6364 29 27 29C33.3636 29 37 26 37 26"
+    stroke="black"
+    stroke-width="2"
+  />
+</svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 54 49">
  <path stroke="#000" stroke-width="2" d="M25 47 7.5 39.8a2 2 0 0 1 0-3.6l19-7.9a2 2 0 0 1 1.6 0l18.5 7.9a2 2 0 0 1 0 3.6l-17 7.2a6 6 0 0 1-4.6 0Z"/>
  <path fill="#fff" stroke="#0F1218" stroke-width="2" d="m13.8 36 1 .4-1-.3c-.3 1.1 0 2.5 1.4 3.1 1.8.8 5.4 2 12.2 2a31 31 0 0 0 12.2-2c1.4-.5 1.7-2 1.3-3.2L29.8 5a3 3 0 0 0-5.7.2L13.8 36Z"/>
  <path stroke="#000" stroke-width="2" d="M21.8 14.8s2 1.4 5.4 1.4c3.4 0 5.4-1.4 5.4-1.4M17 26s3.6 3 10 3 10-3 10-3"/>
</svg>

```

Reply:
I copy pasted your changes but prettier changes them a bit further. Lmk if it looks good to you. (Also, I'm assuming that there is something in the angular-build pipeline that does image optimization/compression for us?)